lib/backend/model/bgppeer.go

```diff
@@ -63,12 +65,13 @@ func (key NodeBGPPeerKey) valueType() (reflect.Type, error) {
 }
 
 func (key NodeBGPPeerKey) String() string {
-	return fmt.Sprintf("BGPPeer(node=%s, ip=%s)", key.Nodename, key.PeerIP)
+	return fmt.Sprintf("BGPPeer(node=%s, ip=%s, keep_next_hop=%s)", key.Nodename, key.PeerIP, strconv.FormatBool(key.KeepOriginalNextHop))
 }
 
 type NodeBGPPeerListOptions struct {
-	Nodename string
-	PeerIP   net.IP
+	Nodename            string
+	PeerIP              net.IP
+	KeepOriginalNextHop bool
```

Comment:
Same for the list options, the new field should only go in the value type.

Reply:
In deed! It is unnecessary, I'll remake up